test/e2e/framework/util.go

```diff
@@ -552,6 +552,9 @@ func LoadConfig() (config *restclient.Config, err error) {
 	}
 	c, err := restclientConfig(TestContext.KubeContext)
 	if err != nil {
+		if TestContext.Host == "" {
+			TestContext.Host = c.Clusters[c.CurrentContext].Server
```

Comment:
+1 on explaining in which tests are you seeing failures?
also if `err != nil`, `c` will probably `nil` which means this line will panic.

Reply:
I added more checkings here. PTAL.